test/www/jxcore/test/thalicryptomanagertests.js

```diff
@@ -0,0 +1,61 @@
+require('./mockmobilegetdocpath');
+var test = require('tape');
+var cryptomanager = require('../thali/thalicryptomanager');
+
+
+test('successfully create a new pkcs12 file and return hash value', function (t) {
+  var errorMessage = null,
+      fileLocation = './pkcs12foldergood';
```

Comment:
The folder where the pkcs12 file will be saved will be created before every test begins and deleted at the end of every test.